### ISSUE TITLE
`simp [-foo]` missing go to def

### ISSUE THREAD

**Issue by digama0** [leanprover/lean4#3724]
```lean
example : 0 + x = x + 0 := by
  simp [-Nat.zero_add]; exact Nat.zero_add x
```
Ctrl-clicking on `Nat.zero_add` doesn't work. (I will submit a PR for this if no one beats me to it.)

**Comment by semorrison** [comment]
Happy to merge such a PR!

**Comment by leodemoura** [comment]
Fixed by #4176
